src/local_storage_store.rs

```diff
@@ -81,4 +83,10 @@ impl StoreImpl for LocalStorageStore {
         storage.set_item(&key, &json).map_err(SetError::SetItem)?;
         Ok(())
     }
+
+    fn clear(&mut self) -> Result<(),SetError> {
+        let storage = self.storage();
+        storage.clear().map_err(SetError::Clear)?;
```

Comment:
This is potentially confusing when multiple apps on the same domain access local storage.

The API here has scopes, but calling clear in one store will clear all the data in the others as well.

Perhaps it's possible to iterate over the keys and only delete items if the prefix matches?

Reply:
thank you i will reconsider this situation